# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../pida/hotfix.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./pida/services/commander/commander.py	(revision 1109)
+++ ./pida/services/commander/commander.py	(working copy)
@@ -79,13 +79,13 @@
             _('Whether a background image will be displayed'),
         )
 
-        self.create_option(
-            'background_image_file',
-            _('Background image file'),
-            OTypeFile,
-            '',
-            _('The file used for the background image'),
-        )
+        #self.create_option(
+        #    'background_image_file',
+        #    _('Background image file'),
+        #    OTypeFile,
+        #    '',
+        #    _('The file used for the background image'),
+        #)
 
         self.create_option(
             'cursor_blinks',
